src/rpc.ts

```diff
@@ -81,7 +81,7 @@ router.post('/', async (req, res) => {
       return rpcError(res, 'UNAUTHORIZED', id);
     }
   } catch (e: any) {
-    capture(e, { context: { body: req.body } });
+    capture(e, { contexts: { body: req.body } });
```

Comment:
in other places, you write
```
capture(e, { contexts: { input: { body: req.body } } });
```
but here
```
capture(e, { contexts: { body: req.body } });
```
why does it have a different structure?

Reply:
fixed

Reply:
still the question about hiding the nested structure of the sentry object. It requires some changes in `snapshot-sentry` library. The idea is to simplify the interface of capture from `capture(e, { contexts: { input: { body: req.body } } });` to `capture(e, { body: req.body });`.

Reply:
> We allow the following context keys to be passed: tags, extra, contexts, user, level, fingerprint.

By hiding the `contexts` key, we lose the ability to set it to something else, like `tags` etc ...

Reply:
Do we set `tags` in any service? We should keep it simple.

Reply:
Not at the moment, but by abstracting it, we remove the option to customize the context later. It'll also make the `capture` signature different from Sentry own `captureException`, which make it more difficult to refactor in case we can to use Sentry own capture directly